website/docs/faqs/Snapshots/snapshot-target-is-not-a-snapshot-table.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Debug Snapshot target is not a snapshot table errors"
+description: "Debugging Snapshot target is not a snapshot table"
+sidebar_label: "Snapshot target is not a snapshot table"
+id: snapshot-target-is-not-a-snapshot-table
+---
+
+If you see the following error when trying to snapshot:
+
+> Snapshot target is not a snapshot table (missing `dbt_scd_id`, `dbt_valid_from`, `dbt_valid_to`)
+
+Double check that you haven't inadvertently caused your snapshot to behave like table materializations by setting its `materialized` config to be `table`. Prior to dbt version 1.4, it was possible to have a snapshot like this:
+
+```sql
+{% snapshot snappy %}
+  {{ config(materialized = 'table', ...) }}
+  ...
+{% endsnapshot %}
+```
+
+dbt is treating snapshots like tables (issuing `create or replace table ...` statements) **silently** instead of actually snapshotting data (SCD2 via `insert` / `merge` statements). When upgrading to dbt versions 1.4 and higher, dbt now raises a Parsing Error (instead of silently treating snapshots like tables) that reads:
+
+```
+A snapshot must have a materialized value of 'snapshot'
+```
+
+This tells you to change your `materialized` config to `snapshot`. However, as soon as you do that, you may run into the error above (missing "dbt_scd_id", etc). This is because, previously, when dbt treated snapshots like tables - there were no [snapshot meta-fields](/docs/build/snapshots#snapshot-meta-fields) added to your snapshot target table - and because those meta-fields don't exist, dbt correctly identifies and tells you that you intend to snapshot into a table that isn't a snapshot.
```

Comment:
```suggestion
This tells you to change your `materialized` config to `snapshot`. But when you make that change, you might encounter an error message saying that certain fields like `dbt_scd_id` are missing. This error happens because, previously, when dbt treated snapshots as tables, it didn't include the necessary [snapshot meta-fields](/docs/build/snapshots#snapshot-meta-fields) in your target table. Since those meta-fields don't exist, dbt correctly identifies that you're trying to create a snapshot in a table that isn't actually a snapshot.
----
This tells you to change your `materialized` config to `snapshot`. However, as soon as you do that, you may run into the error above (missing "dbt_scd_id", etc). This is because, previously, when dbt treated snapshots like tables - there were no  added to your snapshot target table - and because those meta-fields don't exist, dbt correctly identifies and tells you that you intend to snapshot into a table that isn't a snapshot.
```